vissl/data/ssl_transforms/img_pil_to_rgb_mode.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
I think we need to make the transform name more specific otherwise users might mistakenly start using this transform for non-mnist datasets.

How about `img_pil_mnist_to_rgb_mode.py` ? Unless we think there will be more generic use of this transform, we can mention mnist specifically. Thoughts?

Reply:
Yes, I agree with you, I will rename it accordingly :)

Reply:
It should now be fixed in 1d0d4a1162346fde86938fa5f9e53291ef55f55e